### DISCUSSION TITLE
Dockerfile??

### DISCUSSION THREAD

**Discussion by Kannan665** [open-edge-platform/anomalib#1140] [Q&A] [answered]
As per the documentation in "https://openvinotoolkit.github.io/anomalib/getting_started/installation/docker/index.html#".....

"Build the Docker Image
To use anomalib with Docker, you can build a Docker image containing anomalib all its dependencies from the provided Dockerfile. To this end, navigate to the anomalib root directory (the one containing the Dockerfile) and build the Docker image with"

I could not find the Dockerfile.

**Comment by samet-akcay** [comment, marked as answer]
It is here 
https://github.com/openvinotoolkit/anomalib/blob/main/.ci/Dockerfile

**Comment by Kannan665** [comment]
> It is here https://github.com/openvinotoolkit/anomalib/blob/main/.ci/Dockerfile

Thank you. My ubuntu file explorer was not showing it up, as it was inside a ".ci" folder and it was hidden. I got it now......

**Comment by aboccag** [comment]
Hello,

Has it been removed ?

Best regards

**Reply by ashwinvaidya17** [reply]
Yes, we removed the dockerfile. It wasn't always up-to-date, and we didn't have a significant number of community members using it.
